Sources/OktaOidc/macOS/Internal/OktaOidcSignOutHandlerMAC.swift

```diff
@@ -38,13 +38,12 @@ class OktaOidcSignOutHandlerMAC: OktaOidcSignOutHandler {
         oidcClient.signOutOfOkta(authStateManager: authStateManager,
                                  redirectServerConfiguration: redirectServerConfiguration) { error in
             notFinishedOptions.remove(.signOutFromOkta)
-            if let _ = error {
+            
+            if error != nil {
                 failedOptions.insert(.signOutFromOkta)
             }
-            super.signOut(with: notFinishedOptions,
-                          failedOptions: failedOptions,
-                          progressHandler: progressHandler,
-                          completionHandler: completionHandler)
+            
+            super.signOut(with: notFinishedOptions, failedOptions: failedOptions, progressHandler: progressHandler, completionHandler: completionHandler)
```

Comment:
The multiline version was easier to read

Reply:
Resolved.